xkcd/cmd/main.go

```diff
@@ -0,0 +1,42 @@
+// xkcd is an xkcd offline indexing and quering tool.
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/vespian/go-exercises/xkcd/pkg/cmdline"
```

Comment:
JDef suggested me this approach and gave Kubernetes as an example of a project using it.

The thing that I am afraid of here is that the more packages you have the more difficult is to navigate main projects directory. I have this problem for i.e. dcos-ca.

This is just a toy project but I would like to develop good habits as soon as possible :)